robottests/translation.txt

```diff
@@ -29,6 +29,7 @@ Can switch to other language  [Arguments]  ${button}  ${language}
 Enter Translation
     Open Browser  ${BASE_URL}  ${BROWSER}
     Click Link  Translations
+    Set Browser Implicit Wait     0.2 seconds
```

Comment:
Yup, but sleeps are unreliable and make tests slow, so you'll want to poll instead. In Robot, there should be Wait Until keywords which poll. Then you can wait until image is loaded, with a timeout. That ought to be better.